css-positioning/fixed.css

```diff
@@ -0,0 +1,13 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+
+  /* your code here */
+  position: fixed;
+  left: 640px;
```

Comment:
Rather than this large movement, you should just make this take the full width available, and the `text-align-center` can do the work of centering the text